ts/screens/onboarding/EmailInsertScreen.tsx

```diff
@@ -117,35 +132,61 @@ class EmailInsertScreen extends React.PureComponent<Props, State> {
     });
   }
 
+  private isEditing = this.props.navigation.getParam("isEditing") || false;
+
+  public componentDidMount() {
+    if (this.isEditing) {
+      this.setState({ email: some(EMPTY_EMAIL) });
+    }
+  }
+
   public render() {
+    const { isEditing } = this;
     return (
       <BaseScreenComponent
-        goBack={this.handleGoBack}
-        headerTitle={I18n.t("onboarding.email.insert.headerTitle")}
+        goBack={() =>
+          isEditing ? this.props.navigation.goBack() : this.handleGoBack()
```

Comment:
Try to change this lamba function to a const

Reply:
a new approach is proposed here: https://github.com/teamdigitale/io-app/pull/1363/commits/2d1442691356a14fc1b5864a0a30220da184f8cb